backend/models/discount.js

```diff
@@ -34,6 +38,15 @@ module.exports = (sequelize, DataTypes) => {
       },
       uses: {
         type: DataTypes.INTEGER
+      },
+      data: {
+        type: isPostgres ? DataTypes.JSONB : DataTypes.JSON
+      },
+      minCartValue: {
```

Comment:
nit: maybe add a comment explaining in what unit this value is. Is it in cents? Same thing for maxDiscountValue below

Reply:
Yep, it's in cents. Will add a comment